frontend/express/public/core/events/stylesheets/_main.scss

```diff
@@ -53,38 +51,97 @@
         font-family: t.$font-family;
         color: #333c48;
     }
-    &--green {
-        color: #12AF51;
-        padding-left: 8px;
-        font-size: 14px;
+    &__trend {
         font-weight: 500;
         line-height: 14px;
     }
-    &--red {
-        padding-left: 8px;
-        color: red;
-        font-size: 14px;
-        font-weight: 500;
-        line-height: 14px;
+    &__arrow {
+        padding-right: 3px;
     }
 }
 .cly-vue-events-overview-top-events {
     padding-left: 24px;
     padding-bottom: 16.5px;
+    &--tooltip {
+        margin-left: 5.5px;
+    }
+    &__description {
+        font-weight: 500;
+        line-height: 14px;
+    }
+    &__trend {
+        padding-right: 3px;
+    }
 }
 .cly-vue-events-overview-monitor-events {
+    &__charts{
+    .cly-vue-chart__echart{
+        padding: 0px;
```

Comment:
why this? 😄 
afaik, there is no chart in manage events.
is there?

Reply:
There is. This is for monitor-events on overview page. :)

Reply:
couldn't find it.
can you share a link or a screenshot.

Reply:
Please find link below.
https://abhishek.count.ly/dashboard#/60a93d3669c7e542faaa5a27/analytics/events/overview